src/components/PlaceItem/PlaceItem.tsx

```diff
@@ -0,0 +1,34 @@
+// import Image from "next/image";
+import { Place } from "../../data/memorialPlacesData";
+import { Button } from "../shared/Button/Button";
```

Comment:
Use absolute imports (`"@/components/etc"`) if it is a cross module import (or at all)